lib/SIL/SILInstructions.cpp

```diff
@@ -1061,6 +1061,30 @@ bool TermInst::isFunctionExiting() const {
   llvm_unreachable("Unhandled TermKind in switch.");
 }
 
+bool TermInst::isProgramTerminating() const {
+  switch (getTermKind()) {
+    case TermKind::BranchInst:
```

Comment:
Did you run this through git-clang-format?

Reply:
Ah! Thanks, I will. Xcode's reindent-on-paste indents each case 😢